source/_docs/add-client-site.md

```diff
@@ -22,3 +22,23 @@ While many of these steps are followed through the normal course of an Agency-Cl
 The Agency should follow these steps:
 
 {% include("content/invite-to-pay.html")%}
+
+## Accepting an Invitation to Pay
+1. When an agency is ready to transfer ownership of the site to you, you'll receive an email inviting you to pay for the site:
```

Comment:
Possibly use "supporting organization" instead of agency, because it might not always be an agency... and that would be more consistent to how we refer that relationship elsewhere (unless that has also changes recently).

Reply:
Agreed! Resolved in https://github.com/pantheon-systems/documentation/pull/4556/commits/b4d6a5c84c9dd6d9c43da4bb8049d4afb8555447